src/element/textElement.ts

```diff
@@ -324,6 +324,10 @@ export const getTextHeight = (text: string, font: FontString) => {
 };
 
 export const wrapText = (text: string, font: FontString, maxWidth: number) => {
+  if (!maxWidth) {
+    return text;
+  }
```

Comment:
This looks like a hacky fix. The `width` of the element should never be computed to `NaN`, can we instead make sure that width is never set to `NaN` ?
cc @dwelle

Reply:
I believe that's what the collision fix was for. But there may be other cases where we get NaN, so we should definitely make sure we don't get stuck in an infinite loop here. Either we use a fallback for `maxWidth` here, or we return `text` unwrapped as @zsviczian is doing, which is fine, considering.

Reply:
Alright in that case we should add a spec as well. I can add the spec and then merge.